docs/items/tools.md

```diff
@@ -0,0 +1,246 @@
+public static final Supplier<ShovelItem> COPPER_SHOVEL = ITEMS.register("copper_shovel", () -> new ShovelItem(...));
+public static final Supplier<HoeItem> COPPER_HOE = ITEMS.register("copper_hoe", () -> new HoeItem(...));
+```
+
+### Tags
+
+When creating a `Tier`, it is assigned a block tag containing blocks that need this tool (or a better one) to be broken. For example, the `minecraft:needs_iron_tool` tag contains Diamond Ores (among others), and the `minecraft:needs_diamond_tool` tag contains blocks like Obsidian and Ancient Debris.
+
+You can reuse one of these tags for your tool if you're fine with that. For example, if we wanted our copper tools to just be more durable stone tools, we'd pass in `BlockTags.NEEDS_STONE_TOOL`.
+
+Alternatively, we can create our own tag, like so:
+
+```java
+public static final TagKey<Block> NEEDS_COPPER_TOOL = TagKey.create(BuiltInRegistries.BLOCK.key(), new ResourceLocation(MOD_ID, "needs_copper_tool"));
+```
+
+And then, we populate our tag. For example, let's make copper able to mine gold ores, gold blocks and redstone ore, but not diamonds or emeralds. (Redstone blocks are already mineable by stone tools.) The tag file is located at `src/main/resources/data/mod_id/tags/blocks/needs_copper_tool.json` (where `mod_id` is your mod id):
+
+```json
+{
+  "values": [
+    "minecraft:gold_block",
+    "minecraft:raw_gold_block",
+    "minecraft:gold_ore",
+    "minecraft:deepslate_gold_ore",
+    "minecraft:redstone_ore",
+    "minecraft:deepslate_redstone_ore"
+  ]
+}
+```
+
+Finally, we can pass our tag into our tier creation, as seen above.
+
+### `TierSortingRegistry`
+
+In order to make the game actually pick up your tier as between two others, you must register it to the `TierSortingRegistry`. This must happen before item registration, a `static` initializer in the same class as your tier definition is a good spot for that. If you do not add your tier to the registry, it will fall back to what vanilla would do.
+
+```java
+public static final Tier COPPER_TIER = new SimpleTier(...);
+
+static {
+    TierSortingRegistry.registerTier(
+            COPPER_TIER,
+            //The name to use for internal resolution. May use the Minecraft namespace if appropriate.
+            new ResourceLocation("minecraft", "copper"),
+            //A list of tiers that are considered lower than the type being added. For example, stone is lower than copper.
+            //We don't need to add wood and gold here because those are already lower than stone.
+            List.of(Tiers.STONE),
+            //A list of tiers that are considered higher than the type being added. For example, iron is higher than copper.
+            //We don't need to add diamond and netherite here because those are already higher than iron.
+            List.of(Tiers.IRON)
+    );
+}
+```
+
+If you want to check if a tier is applicable for a block state, you can call `TierSortingRegistry#isCorrectTierForDrops`.
+
+## Custom Tool Types
```

Comment:
Multitools (pickaxe, axe, shovel, hoe) typically extend the `AxeItem` class because of enchantments for the axe being hardcoded to a class check. Not sure if this should be in here, but I figured it's useful info.